Helper.java

```diff
@@ -99,6 +108,35 @@ static void enterData() throws InvalidAnimalName {
                     }
                 }
                 case 3 -> {
+                    for (Map.Entry<String, List<Animal>> mp : animalSpecies.entrySet()) {
+                        if (mp.getKey().equals("dog")) {
+                            System.out.println(mp.getValue());
+                        }
+                    }
+                }
+                case 4 -> {
+                    for (Map.Entry<String, List<Animal>> mp : animalSpecies.entrySet()) {
+                        if (mp.getKey().equals("cat")) {
+                            System.out.println(mp.getValue());
+                        }
+                    }
+                }
+                case 5 -> {
+                    for (Map.Entry<String, List<Animal>> mp : animalSpecies.entrySet()) {
+                        if (mp.getKey().equals("horse")) {
+                            System.out.println(mp.getValue());
+                        }
+                    }
+                }
+                case 6 -> {
+                    System.out.println("Please enter a color of instances to display (brown, white, black): ");
```

Comment:
A little inconvinient for user to type words let`s replace it with numbers
1 - brown etc.